src/tools/crawling_tools.py

```
@@ -991,34 +1020,201 @@ async def _process_code_examples(ctx: Context, crawl_results: List[Any], directo
+    
+    Args:
+        file_path: Path to the file
+        
+    Returns:
+        File type classification
+    """
+    file_ext = os.path.splitext(file_path)[1]
+    
+    if file_ext in ['.py', '.js', '.ts', '.java', '.cpp', '.c', '.go', '.rs']:
+        return "SourceCode"
+    elif file_ext in ['.md', '.rst', '.txt']:
```

Ruff (0.12.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)